portfolio.html

```diff
@@ -0,0 +1,231 @@
+            </div>
+            <p class="tonic">
+              A daily selection of privately personalized reads; no accounts or sign-ups required.
+            </p>
+            <div class="tags tonic">
+              <ul>
+                <li>html</li>
+                <li>css</li>
+                <li>javaScript</li>
+              </ul>
+              <button type="button"><a href="">See Project</a></button>
+            </div>
+        </div>
+      </div>
+      <div class="item2">
+        <div class="item2a">
+          <img src="images/Snapshoot%20Portfolio2.png">
+        </div>
+        <div class="item1b">
+          <h2>Multi-Post Stories</h2>
+          <div class="frame2">
+           <span>FACEBOOK</span>
+            <ul class="role">
+              <li>Full Stack Dev</li>
+              <li>2015</li>
+            </ul>
+          </div>
+          <p >
+            Experimental content creation feature that allows users to add to an existing story over the course of a day without spamming their friends.
+          </p>
+          <div class="tags">
+            <ul>
+              <li>html</li>
+              <li>css</li>
+              <li>javaScript</li>
+            </ul>
+            <button type="button"><a href="">See Project</a></button>
+          </div>
+        </div>
+      </div>
+      <div class="item3">
+        <div class="item3a">
+          <img src="images/Snapshoot%20Portfolio3.png" alt="" />
+        </div>
+        <div class="item1b">
+          <h2 class="facebook">Facebook 360</h2>
+          <div class="frame2 facebook">
+            <span>FACEBOOK</span>
+            <ul class="role">
+              <li>Full Stack Dev</li>
+              <li>2015</li>
+            </ul>
+          </div>
+          <p class="facebook">
+            Experimental content creation feature that allows users to add to an existing story over the course of a day without spamming their friends.
+          </p>
+          <div class="tags facebook">
+            <ul>
+              <li>html</li>
+              <li>css</li>
+              <li>javaScript</li>
+            </ul>
+            <button type="button"><a href="">See Project</a></button>
+          </div>
+        </div>
+      </div>
+      <div class="item4">
+        <div class="item4a">
+          <img src="images/Snapshoot%20Portfolio4.png" alt="" />
+        </div>
+        <div class="item1b">
+          <h2>Uber Navigation</h2>
+          <div class="frame2">
+            <span>Uber</span>
+            <ul class="role">
+              <li>Lead Developer</li>
+              <li>2015</li>
+            </ul>
+          </div>
+          <p>
+           Experimental content creation feature that allows users to add to an existing story over the course of a day without spamming their friends.
+          </p>
+          <div class="tags">
+            <ul>
+              <li>html</li>
+              <li>css</li>
+              <li>javaScript</li>
+            </ul>
```

Comment:
- [ ] please consider increasing the padding, font-size, font-family of language to match the figma design as mentioned by the previous reviewer. 

**Your design**
![r8](https://user-images.githubusercontent.com/74411135/229213967-48d5004b-588d-4c77-bdd9-a16783d8452e.png)

**Figma design**
![image](https://user-images.githubusercontent.com/74411135/229214049-2e884653-7302-4a8a-91b8-546356118308.png)

This will make your design prettier which will improve the user experience 📱 💻 🖥️ 😃 😺